### DISCUSSION TITLE
Bold figure "kind"

### DISCUSSION THREAD

**Discussion by trya2l** [typst/typst#3899] [Q&A]
Hi,

My question is a bit similar to  #3871, but I am just interested to know if it is possible to stylize the "kind" of a figure.

I would like to do something like `#set figure.kind(weight:bold)`

**Comment by cu1ch3n** [comment]
Hi! I don't know why you would like to stylize "kind". "kind" is merely a label indicating the type of figure, not content.

**Reply by trya2l** [reply]
Hi, my teammates had done the captions manually with "Figure" in bold. I wanted to make it correctly without affecting the style. But I am still too unfamiliar with typst to know if it is something I can already modify
